blueprints/addon/additional-dev-dependencies.json

```
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "@embroider/test-setup": "0.36.0",
```

We should probably allow a range here

```suggestion
    "@embroider/test-setup": "^0.36.0",
```